projects/demo/src/modules/components/dialog/examples/10/dialog-example/dialog-example.component.less

```diff
@@ -0,0 +1,60 @@
+.pulldown-menu {
+    background: var(--tui-elevation-01);
```

Comment:
We should use something less radical in terms of styles for this feature. For something this much different to default dialog people should create their own dialogs, not alter built-in appearance with CSS.